cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -60,6 +60,13 @@
       cronjob at:'30 7 * * 1-5', do:deploy_dir('bin', 'cron', 'commit_content')
       cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'cron', 'update_dts')
       cronjob at:'0 17 * * *', do:deploy_dir('bin', 'cron', 'commit_trusted_proxies')
+      # This should be run after the commit_content job that happens on both the levelbuilder
+      # and staging environments
+      # merge_lb_to_staging is also known as DTS or "deploy to staging"
+      cronjob at:'35 7 * * 1-5', do:deploy_dir('bin', 'cron', 'merge_lb_to_staging')
+      # This should be run after the commit_content job that happens on the levelbuilder
+      # environment.
+      cronjob at:'20 9 * * 1-5', do:deploy_dir('bin', 'cron', 'deploy_to_levelbuilder')
```

Comment:
The `if test` and `if levelbuilder` conditionals are no longer wrapping these commands. Will these run in every environment now?

Reply:
Not sure if I did this right, but they should be wrapped by the `if staging` conditional now.

Reply:
I don't see where that is/

Reply:
line 50